helm 5.3: release notes suggestions (#7834)
grafana/mimir: docs/sources/helm-charts/mimir-distributed/release-notes/v5.3.md

- The timeout for writing recording rule results has been increased to 10 seconds to accommodate large result sets.
- The distributor termination grace period was increased from `60s` to `100s` to give more time for a clean termination.
- The read timeout for chunks and index caches using memcached has been increased to from `450ms` to `750ms`.
- For the distributor and query-frontend components the GRPC maximum connection age and the component shutdown delay has been tuned to avoid invoking components that are shutting down.

## Features and enhancements

Notable enhancements are as follows:

- Added experimental feature for deploying [KEDA](https://keda.sh) ScaledObjects as part of the helm chart for the components: distributor, querier, query-frontend and ruler. For more details please see the values sections `kedaAutoscaling`, `distributor.kedaAutoscaling`, `ruler.kedaAutoscaling`, `query_frontend.kedaAutoscaling`, and `querier.kedaAutoscaling`, and the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).

> - Gateway: Allow to configure whether or not NGINX binds IPv6 via `gateway.nginx.config.enableIPv6`.

Reviewer: nit: "allow to configure" isn't traditionally standard English.

I know that your patch doesn't introduce this change so feel free to ignore the suggestion.
I've tried to rewrite it so that it is parallel to the previous entry in terms of tense and verbiage.
```suggestion
- Gateway: Added option for configuring whether or not NGINX binds IPv6: `gateway.nginx.config.enableIPv6`.
```

Author: thanks Jack!
